README.md

```diff
@@ -1,6 +1,9 @@
 # showmy.chat
+
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+
 [![All Contributors](https://img.shields.io/badge/all_contributors-8-orange.svg?style=flat-square)](#contributors-)
+
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
```

Comment:
This comment block here is managed by the All Contributors bot and, I suspect, could be a frequent game of cat-and-mouse between linter and All Contributors. Let's leave these changes in for now, and if All Contributors breaks or if we find ourselves constantly showing this block in our diffs, we may want to consider ignoring `README` in our Markdown linting.

Reply:
I saw an all-contributers comment somewhere that disabled pretter and markdownlint (even before they were enabled).
I'll add one in here too. That way it doesn't turn into a turf-war between 2 tools